study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+I'm confused about callback functions.
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+Are callback functions functions that only run when a specific condition has been met?
```

Comment:
Callback functions are functions that are passed in as arguments to another function.  Here is an example where ```hello()``` and ```hi()``` are callback functions being passed into ```greet()``` as its first argument.
```javascript
function greet(greeter, name) {
  console.log(`Executing greeter callback with ${name} as the argument.`);

  greeter(name);

  console.log('Greeting finished.');
}

function hello(name) {
  console.log(`Hello, ${name}.`);
}

function hi(name) {
  console.log(`Hi, ${name}.`);
}

greet(hello, 'Louis');
// Output:
// Executing greeter callback with Louis as the argument.
// Hello, Louis.
// Greeting finished.

greet(hi, 'Louis');
// Output:
// Executing greeter callback with Louis as the argument.
// Hi, Louis.
// Greeting finished.
```